site/pages/home/contact-us-slice/form/test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Form from '.';
+
+describe('Contact Us Slice - Form state', () => {
+  it('component renders successfully', () => {
+    const wrapper = shallow(<Form
+      errors={{}}
+      fatalError={false}
+      onSubmit={() => {}}
+    />);
+    expect(wrapper.text()).not.to.contain('Oops! Looks like something went wrong.');
+  });
+
+  it('renders given errors', () => {
+    const wrapper = shallow(<Form
+      errors={{
+        message: 'please write something',
+        contact: 'please fill in contact details',
+      }}
+      fatalError={false}
+      onSubmit={() => {}}
+    />);
+    expect(wrapper.text()).to.contain('please write something');
+    expect(wrapper.text()).to.contain('please fill in contact details');
+  });
+
+  it('contains content for users without js', () => {
+    const wrapper = shallow(<Form
+      errors={{}}
+      fatalError={false}
+      onSubmit={() => {}}
+    />);
+    expect(wrapper.find('noscript')).to.have.length(1);
+    expect(wrapper.find('noscript').text()).to.contain('It seems like your JavaScript is disabled.');
+  });
+
+  it('displays an fatalError occured message', () => {
+    const wrapper = shallow(<Form
+      errors={{}}
+      fatalError
+      onSubmit={() => {}}
+    />);
+    expect(wrapper.text()).to.contain('Oops! Looks like something went wrong.');
```

Comment:
Following up what we discussed in person -
I've improved the tests for the parent component, which covers the area of the functionality that changes the state. These tests check that this component gets handed the props it needs. This component then determines what to do with them. I think we're fully covered now!